src/cswinrt/code_writers.h

```diff
@@ -8066,15 +8069,27 @@ NativeMemory.Free((void*)abiToProjectionVftablePtr);
 
             if (!hasOverridableAttribute)
             {
-                w.write(R"(%
+                // Under normal conditions, we would stop here and just emit a minimal amount of code.
+                // However, if IDIC is requested, just continue normally, but omit the CCW generation.
+                // We know we can safely omit that because it wouldn't have normally be generated.
```

Comment:
>We know we can safely omit that because it wouldn't have normally be generated

Can you clarify this.  Is this because we removed an exclusive to check or because we don't need it for the idic exclusive to scenario?

Reply:
It's because it's not needed, because the IDIC support is only about allowing to do IDIC casts to `exclusive_to` interfaces on RCWs. It doesn't allow _implementing_ those interfaces on managed types to then marshal them to native as a CCW. Nothing's stopping us from also doing that, but I just didn't include it here because (1) it increases codegen and (2) it's technically a separate feature and not needed for this scenario. Thoughts?

Reply:
Makes sense, for this scenario if this flag is being used to project an interface, it doesn't need it.  Just wanted to confirm if you were thinking the same.

Reply:
Yeah, the rationale being that if the interface is `exclusive_to`, it doesn't really make sense for anyone else to implement it (because that's the whole point of it being exclusive after all). But it's still completely valid to just do an IDIC cast to it on an object (which _can_ implement it natively), so you can invoke its methods (which is what we're doing) 🙂